util-taglib/src/com/liferay/taglib/ui/AppViewEntryTag.java

```diff
@@ -62,7 +62,7 @@ public void setAssetTagClassPK(long assetTagClassPK) {
 	}
 
 	public void setAuthor(String author) {
-		_author = author;
+		_author = HtmlUtil.unescape(author);
```

Comment:
@daviddotzhang @daledotshan 

It doesn't make sense that we add an `unescape` here in the `.java` file and then turn around and do an escape in the `.jsp`.

Also, this change introduces a bug in `/apps/item-selector/item-selector-taglib/src/main/resources/META-INF/resources/repository_entry_browser/page.jsp`